src/main/charts/bamboo/templates/statefulset.yaml

```diff
@@ -74,15 +74,26 @@ spec:
           command: ["sh", "-c", {{ include "bamboo.sharedHome.permissionFix.command" . | quote }}]
         {{- end }}
         {{- include "common.jmx.initContainer" . | nindent 8 }}
-        {{- if .Values.bamboo.additionalCertificates.secretName }}
+        {{- if or .Values.bamboo.additionalCertificates.secretName .Values.bamboo.additionalCertificates.secretList }}
         - name: import-certs
           image: {{ include "bamboo.image" . | quote }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           volumeMounts:
             - name: keystore
               mountPath: /var/ssl
+          {{- if.Values.bamboo.additionalCertificates.secretName }}
             - name: certs
               mountPath: /tmp/crt
+          {{- else }}
+          {{- range .Values.bamboo.additionalCertificates.secretList }}
+            {{- $secretName := .name }}
+            {{- range .keys }}
+            - name: {{ $secretName }}
```

Comment:
Correct me if I'm wrong, if there are 2 certs in this secret, it'll ended up having 2 volumeMounts having `- name: {{ $secretName }}` ?

Reply:
which will not be favoured by K8s? If I remember correctly, each volumeMount name must be unique within a container.

Reply:
Ohh, that's the volume name, and will have different sub path. Ignore the above comment, sorry.